graphql/task_resolver.go

```diff
@@ -100,15 +100,15 @@ func (r *taskResolver) BaseStatus(ctx context.Context, obj *restModel.APITask) (
 func (r *taskResolver) BaseTask(ctx context.Context, obj *restModel.APITask) (*restModel.APITask, error) {
 	t, err := obj.ToService()
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting service model for APITask %s: %s", *obj.Id, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting service model for APITask %s: %s", *obj.Id, err.Error()))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting service model for APITask '%s': %s", *obj.Id, err.Error()))
```